apps/bc-wallet-api-server/src/database/migrations/0006_bc-wallet-api.sql

```diff
@@ -1,4 +1,6 @@
 ALTER TABLE "credentialDefinition" ALTER COLUMN "icon" DROP NOT NULL;--> statement-breakpoint
 ALTER TABLE "stepAction" ADD COLUMN "credential_definition_id" text;--> statement-breakpoint
 ALTER TABLE "stepAction" ADD COLUMN "connection_id" text;--> statement-breakpoint
-ALTER TABLE "stepAction" ADD COLUMN "go_to_step" text;
\ No newline at end of file
+ALTER TABLE "stepAction" ADD COLUMN "go_to_step" text;--> statement-breakpoint
```

Comment:
why are we changing a migration?

Reply:
It comes from @sanderPostma's code. I've added the --> statement-breakpoint and for some reason it shows the entire line as new